registry/Scripts/self-signed.ps1

```diff
@@ -1,13 +1,27 @@
 #requires -RunAsAdministrator
 
-# Password and Common Name of the certificate
-$PASSWORD=""
-$CN=""
-# Export path of the certificate
-$PATH=""
+Param
+(
+    [Parameter(Mandatory = $true, HelpMessage = "Common DNS FQDN name to be used .")]
+    [string] $DNSCName,
```

Comment:
`CertificateCN` may be a better name
whoever wrote `New-SelfSignedCertificate` picked param names poorly

Reply:
Updated.